File: src/main/java/com/example/ssauc/user/bid/dto/ReportDto.java
package com.example.ssauc.user.bid.dto;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ReportDto {
    private Long productId;
    private Long reporterId;
    private Long reportedUserId;
    private String reportReason;
    private String details;



}

File: src/main/java/com/example/ssauc/user/bid/dto/ReportDto.kt
package com.example.ssauc.user.bid.dto

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
class ReportDto {
    public val productId: Long? = null
    public val reporterId: Long? = null
    public val reportedUserId: Long? = null
    public val reportReason: String? = null
    public val details: String? = null
}
